dvc/repo/live.py

```diff
@@ -31,8 +32,9 @@ def create_summary(out):
     index_path = render(
         out.repo, renderers, metrics=metrics, path=html_path, refresh_seconds=5
     )
-
-    webbrowser_open(index_path)
+    auto_open = out.repo.config["plots"].get("auto_open", False)
+    if auto_open and int(os.environ.get(DVCLIVE_OPEN, "1")):
```

Comment:
Use `utils.env2bool` for this, it covers more of the commonly used true/false-y env var cases besides `"1"`

https://github.com/iterative/dvc/blob/30d6839a5f471fb02a7a2dcbd6a6616b71277762/dvc/utils/__init__.py#L335